src/ansys/dpf/composites/data_sources.py

```diff
@@ -75,13 +75,25 @@ def __init__(
             True if files are on the local machine, False if they have already
             been uploaded to the DPF server..
         """
-        if isinstance(rst, (str, pathlib.Path)):
-            rst = [rst]
-        self.rst = rst  # type: ignore
+        self.rst = self._get_rst_list(rst)
```

Comment:
Do we need the explicit `_get_rst` here if it's already handled in the `__setattr__`?

Reply:
No, it's not needed. Thanks.